src/UnitySampleProject/Assets/Samples/web3.unity SDK/3.0.4/Web3.Unity Samples/Scripts/Samples/EvmSample.cs

```diff
@@ -126,7 +126,7 @@ public async Task<string> ContractCall()
     {
         object[] args =
         {
-            Web3Unity.Web3.Signer.PublicAddress
+            Web3Unity.Instance.Address
```

Comment:
I'd say `Address` is not descriptive enough as it can mean any address in the context of Web3Unity. Something like `PlayerAddress` is more suiting imo.

Reply:
Hmm I suppose yeah, this makes me think I don't think Web3Unity supports connecting multiple acoounts @kantagara what do you think of this as a feature?

Reply:
Maybe it should be called PublicAddress. 
In regards to multiple accounts -> That is something I think we will have to support for our upcoming partnership with you guys know who, so it will be a nice integration. Wouldn't like to use it by default tho in the main branch so to say :D

Reply:
The name hasn't changed

Reply:
Yep my bad, changed now